internal/Picker/Picker.js

```diff
@@ -776,11 +776,11 @@ const PickerBase = class extends ReactComponent {
 	}
 
 	initRef (prop) {
-		return (ref) => {
+		return (componentRef) => {
```

Comment:
The change you've made here is... just the same as before it's only changed the name from `ref` to `componentRef`.
And also, if I put console.log above https://github.com/enactjs/moonstone/blob/master/EditableIntegerPicker/EditableIntegerPickerDecorator.js#L162,
the `this.pickerNode` gives `undefined`.

Here, the `ref` is an HTML DOM element(in this case) that React passes and can be stored and accessed.
Please refer to https://reactjs.org/docs/refs-and-the-dom.html#callback-refs for more details.
So what I expect was to pass this `ref` to `props.componentRef` function that `EditableIntegerPicker` passed.
If the ref is well passed, `this.pickerNode` should be the EditableIntegerPicker DOM node.
Please re-check.

Reply:
You are right. I got confused about the picker being refocused after VKB closed.
I modified the implementation. Now the pickerNode is not undefined.
Also I tested locally on the O20 board with a modified sampler (Button, EditableIntegerPicker, Button) the focus after VKB closing. Seems to work OK